etna/datasets/tsdataset.py

```diff
@@ -576,7 +576,7 @@ def isnull(self) -> pd.DataFrame:
         """
         return self.df.isnull()
 
-    def head(self, n_rows: Optional[int] = None) -> pd.DataFrame:
+    def head(self, n_rows: int = 5) -> pd.DataFrame:
```

Comment:
looks like `None` is a valid value, isn't it?
```suggestion
    def head(self, n_rows: Optional[int] = 5) -> pd.DataFrame:
```